packages/connectors/coinbase-wallet/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@usedapp/coinbase-connector",
+  "version": "0.0.1",
+  "main": "dist/src/index.js",
+  "types": "dist/src/index.d.ts",
+  "repository": "git@github.com:EthWorks/useDApp.git",
+  "author": "Ethworks",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@coinbase/wallet-sdk": "^3.3.0"
```

Comment:
Have you considered pinning exact version? We will have one less thing to consider when debugging bug reports.